airflow/config_templates/airflow_local_settings.py

```diff
@@ -48,6 +49,8 @@
 
 COLORED_FORMATTER_CLASS: str = conf.get_mandatory_value('logging', 'COLORED_FORMATTER_CLASS')
 
+DAG_PROCESSOR_LOG_TARGET: bool = conf.get_mandatory_value('scheduler', 'DAG_PROCESSOR_LOG_TARGET')
```

Comment:
Should this belong to `scheduler` or `logging`?

Reply:
Yeah I'm uncertain of the convention here. It seems like `logging` makes more sense, however, when the logs are sent to files, the proper directory is defined under `scheduler.child_process_log_directory`. That's why I put it in scheduler.

With that in mind, which do you think makes more sense?

Reply:
I’m not sure either, things are a bit all over the place, there’s even a DAG-processor-related config in `core`. I would probably put it under `logging` that’s where most scheduler-related logging configs go.

Reply:
Ok cool, that works for me. Thanks TP!

Reply:
We tried to consolidate all the logging configs into the logging section a couple of years back: https://github.com/apache/airflow/commit/046b36fab7d1841be63d48758eb2a6a197efaf1c

So that's the goal at least, other configs are marked for deprecated

https://github.com/apache/airflow/blob/d82436b382c41643a7385af8a58c50c106b0d01a/airflow/configuration.py#L173-L196

Reply:
so all the logging related configs should be moved there.

```suggestion
DAG_PROCESSOR_LOG_TARGET: bool = conf.get_mandatory_value('logging', 'DAG_PROCESSOR_LOG_TARGET')
```